use-cases/using-Intel-QAT-and-kata.md

```diff
@@ -0,0 +1,491 @@
+# Table of Contents
+
+- [Introduction](#introduction)
+  * [Helpful Links before starting](#helpful-links-before-starting)
+  * [Steps to enable Intel QAT in Kata Containers](#steps-to-enable-intel-qat-in-kata-containers)
+  * [Script variables](#script-variables)
+    + [Set environment variables (Every Reboot)](#set-environment-variables--every-reboot-)
+  * [Prepare the Host (Clear Linux)](#prepare-the-host--clear-linux-)
```

Comment:
This link is incorrect. I'd suggest persevering with https://github.com/kata-containers/tests/tree/master/cmd/check-markdown now that you've resolved the `dos2unix` issue. The link name should be `prepare-the-host-clear-linux`. The `kata-check-markdown check` command will tell you that and indeed `kata-check-markdown toc` will generate a correct TOC for you.

Reply:
I ran the kata-check-markdown to generate the TOC and replaced this one.